test/alternative_spec.rb

```diff
@@ -84,4 +84,65 @@
       end
     end
   end
+
+  describe "#choice" do
+    subject(:alternative) do
+      UpdateAlternatives::Alternative.new(
+        "editor",
+        "auto",
+        "/usr/bin/vim",
+        [
+          UpdateAlternatives::Alternative::Choice.new("/usr/bin/nano", "20", ""),
+          UpdateAlternatives::Alternative::Choice.new("/usr/bin/vim", "30", "")
+        ]
+      )
+    end
+
+    it "changes the alternative's actual choice" do
```

Comment:
maybe append something like `to given parameter` so it is clear how it is changed